### DISCUSSION TITLE
Is it possible to edit the editor's placeholder afterwards?

### DISCUSSION THREAD

**Discussion by GitHubish** [ueberdosis/tiptap#1414] [General Questions]
I am looking to change the editor placeholder based on what is selected by the user in a form. I don't see how to change the placeholder extension configuration. Any ideas? Thanks

**Comment by philippkuehn** [comment]
You can do something like this. Does that help?

```js
Placeholder.configure({
  placeholder: node => {
    if (node.type.name === 'paragraph') {
      return 'placeholder in paragraph'
    }

    return 'placeholder somewhere else'
  },
})
```

**Reply by GitHubish** [reply]
@philippkuehn thank you for your answer. I don't think it helps me. But certainly my need was poorly expressed. 

I initialize an editor with the placeholder for example "Please describe your idea precisely". If the user selects in the form above the editor that it is a bug (radio type button), I would like the placeholder to be now "Please describe your bug with as much information as possible, screenshot ....)

So I would like to be able to configure the Tiptap placeholder extension again to be correct. Should I delete the extension (if possible?) and then add it again with the new configuration? I hope I am clearer on what I want to do.
